### PR TITLE
directly forward seed

### LINKED ISSUE #12: Wrong seed behaviour
The seed is checked simply as `if seed else ...` which results in the same behaviour for seed=None and seed=0, whereas it should be possible to set the seed to anything but `None`: https://docs.python.org/2/library/random.html#random.seed.

Also the seed is stored as member of the class and randomly selected if not None or 0. I would propose to simply forward the seed to `random.Random`. There is no need to store it locally or randomly initialise it outside of the python standard API.

### PR DESCRIPTION
**Pull request by christian-rauch**
This PR removes the manual check of the seed state and will delegate this directly to the random number generator.

Fixes #12 .

### CODE REVIEW

**Comment by christian-rauch** [comment]
I also took the freedom to remove all python versions that are end-of-life (https://devguide.python.org/devcycle/#end-of-life-branches) to speed up CI (and save a tiny bit of energy).

**Comment by christian-rauch** [comment]
@kevinwuhoo Can you have a look at this?

**Comment by kevinwuhoo** [comment]
Wow apologies, I somehow missed the emails on this. Going to merge and cut another release.

File: randomcolor/__init__.py [hunk 1]
```diff
@@ -12,8 +12,7 @@ def __init__(self, seed=None):
         with open(os.path.join(os.path.dirname(__file__), 'lib/colormap.json')) as fh:
             self.colormap = json.load(fh)
 
-        self.seed = seed if seed else random.randint(0, sys.maxsize)
```
Comment (kevinwuhoo) [inline randomcolor/__init__.py:15]: Yeah this line of code makes seems pretty unnecessary in hindsight.

**Comment by kevinwuhoo** [comment]
Thanks for this!

**Comment by kevinwuhoo** [comment]
@christian-rauch I'm sure you have it figured by now but: https://pypi.org/project/randomcolor/0.4.4.6/

**Pull request merged**